src/Avalonia.Base/Controls/Metadata/TemplatePartAttribute.cs

```diff
@@ -51,5 +49,10 @@ public TemplatePartAttribute(string name, Type type)
         /// in <see cref="Name"/>.
         /// </summary>
         public Type Type { get; set; }
+
+        /// <summary>
+        /// Gets or sets flag whether template part is mandatory to be present in the template.  
```

Comment:
"Gets or sets a value indicating whether..." is the correct documentation prefix for bool. Some code analyzers will warn if you don't start the summary docs with this.